Update README.md with process around booking prospect calls (#15534)
fleetdm/fleet: handbook/sales/README.md

    3. Offer training to get them up to speed.
    4. Provide a white glove experience.

> ### Schedule a call with a Solutions Consultant (SC)

Reviewer: ```suggestion
### Schedule a customer call and add a Solutions Consultant (SC)
```

> To schedule an [ad hoc meeting](https://www.vocabulary.com/dictionary/ad%20hoc) with a Fleet customer, the Account Executive will open an issue with [this issue template](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23g-sales%2C%23g-solutions-consultant%2C%3Adiscovery%2C%3Ademo%2C%3Ascoping%2C%3Atech-eval&projects=&template=SC+engagement.md&title=prospect+name+-+prep+%28date%29+-+discovery%2Cdemo%2Cscoping+%28date%29). The Account Executive will need to populate this issue with the appropriate dates to internally prep for the meeting as well as the dates for the external customer meeting. Please allow 1 day lead time between issue creation and actual internal prep meeting. If there is any conflict in scheduling, please feel free to reach out to your friendly neighborhood Solutions Consultant via the #help-solutions-consulting slack channel and we will try to accommodate exceptions where necessary.

Reviewer 2: - idea: inline the github URL instead of an issue template and just have it hard-code the label and the title
- steps for AE: include in handbook "responsibility" and update text here
- `Schedule a call with an SC`
- delete the issue template and use URL instead

    3. Offer training to get them up to speed.
    4. Provide a white glove experience.

> ### Schedule a customer or prospect call and add a Solutions Consultant (SC)

Reviewer: Mike: Should this responsibility be named “Schedule a prospect call”? "Schedule a customer call" would live on the customer success page.

Author: @Sampfluger88 SCs take both pre and post sales calls with prospects and customers.

Reviewer: Agreed, The “Schedule a prospect call” would live on the customer success page.

    3. Offer training to get them up to speed.
    4. Provide a white glove experience.

### Schedule a customer or prospect call and add a Solutions Consultant (SC)
To schedule an [ad hoc meeting](https://www.vocabulary.com/dictionary/ad%20hoc) with a Fleet customer or prospect, the Account Executive (AE) will [open an issue](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23g-sales%2C%23solutions-consultant%2C%3Adiscovery%2C%3Ademo%2C%3Ascoping%2C%3Atech-eval&projects=&template=custom-request.md&title=prospect+name+-+prep+%28date%29+-+discovery%2Cdemo%2Cscoping+%28date%29). 

>  - The AE will populate this issue with the appropriate dates for an internal prep meeting as well as the dates for the external customer meeting.

Reviewer: ```suggestion
```

>  - Secure dates for the external customer meeting whilst consulting the openings available via [this internal calendly](https://calendly.com/fleetdm/talk-to-a-solutions-consultant) (round-robin selection of an available SC). The Account Executive will ensure that there is sufficient lead time to book an internal prep call before the external customer or prospect call. 

Reviewer: ```suggestion
```

>  - Please allow 1 day lead time between issue creation and actual internal prep meeting.

Reviewer: ```suggestion
```

>  - If there is any conflict in scheduling, please feel free to reach out to your friendly neighborhood SC via the #help-solutions-consulting slack channel and we will try to accommodate exceptions where necessary.

Reviewer: ```suggestion
```
